zh/restore-from-s3.md

```diff
@@ -24,8 +24,7 @@ category: how-to
 
 1. 创建 Restore customer resource (CR)，将制定备份数据恢复至 TiDB 集群
 
-+ 创建 Restore custom resource (CR)，通过 AccessKey 和 SecretKey 授权的方式将指定的备份数据恢复至 TiDB 集群：
-
++ 创建 Restore custom resource (CR)，通过 AccessKey 和 SecretKey 授权的方式将指定的备份数据由 Ceph 恢复至 TiDB 集群：
```

Comment:
@ran-huang I'm afraid the format might cause display chaos on the PingCAP website, though it looks well on GitHub. We need to indent four spaces in such cases. I'll update directly.

Reply:
Fixed in https://github.com/pingcap/docs-tidb-operator/commit/49e16e6d354119c2e652e4738b68bbc52d7ed73e.